nodejs/src/wrapper.ts

```diff
@@ -46,11 +46,60 @@ const { NetInstrumentation } = require('@opentelemetry/instrumentation-net');
 const { PgInstrumentation } = require('@opentelemetry/instrumentation-pg');
 const { RedisInstrumentation } = require('@opentelemetry/instrumentation-redis');
 
+// AWS lambda instrumentation response hook for Server-Timing support
+function getEnvBoolean(key: string, defaultValue = true) {
+  const value = process.env[key];
+
+  if (value === undefined) {
+    return defaultValue;
+  }
+
+  if (['false', 'no', '0'].indexOf(value.trim().toLowerCase()) >= 0) {
+    return false;
+  }
+
+  return true;
+}
```

Comment:
```suggestion
  return ['true', 'yes', '1'].includes(value.trim().toLowerCase());
}
```

2 things:

* I think that `includes()` reads slightly cleaner than `indexOf() >= 0`
* I think that defaulting to `true` is a mistake because most defaults are usually false. Opt in vs. opt out. I'd hate for somebody to pass in "never" and get back `true`. 😸

Reply:
Yeah, your intuition serves you well - it's Splunk way for RUM integration. Don't ask me why (as I don't know ;) ).

Reply:
WRT `true` default - it's on purpose, we want to have this enabled unless someone opts out.

Reply:
(it's similar in other places where we have this integration enabled)

Reply:
Opting in a feature is fine, but that should leverage the default value then. I dunno. If you keep this then I wanna see the unit test that says something like `expect(geEnvBoolean('negative')).to.be.true` so that we're all on the same page. 😝

Reply:
It's a pity that we are where we are. If it's enabled by default, then `SPLUNK_TRACE_RESPONSE_HEADER_DISABLED` would be better. Maybe it's a topic for our specification discussion? Empty `SPLUNK_TRACE_RESPONSE_HEADER_ENABLED` interpreted as `true` is not obvious. What if someone sets it to an empty string? I would also be against handling `yes`/`no`, and `0` / `1`, unless our specification already says that we should.

Reply:
Guys, it's an exact copy of the method we have in the official OTel JS distro: https://github.com/signalfx/splunk-otel-js/blob/main/src/options.ts#L257
If it's good enough for the distro, why isn't it good enough for (less critical) lambda?
Also the naming - even if a bit misleading (ENABLE vs DISABLE) - needs to stay consistent between the distro and lambda, that's for sure, right? :)

Reply:
Agree it's a bit of bikeshedding :stuck_out_tongue: 

Personally I haven't liked negations in configuration options (e.g. `X_DISABLED=false` to enable something). `X_ENABLED=true/false` would allow for changing the default behaviour without confusing people, but guess it's a matter of taste. As long as it's documented how to disable it, should be fine either way.

Reply:
Right, I should have mentioned what we discussed internally: I accepted the env extraction code because it's the same what we have in `splunk-otel-js`;